File: 0801_0900/0843/0843.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, m, p, l = (int(x) for x in f.readline().split())
    q.write(f"{m + p + l - n} {n - p - l} {n - m - l}")

File: 0801_0900/0843/0843.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, m, p, l;

	fscanf(f, "%d%d%d%d", &n, &m, &p, &l);
	fprintf(q, "%d %d %d", m + p + l - n, n - p - l, n - m - l);
	return 0;
}
